### PR TITLE
Update Docker/Travis config

### PR DESCRIPTION
**Pull request by thewilkybarkid**
To what we came up with for https://github.com/libero/dummy-api.

### CODE REVIEW

File: docker-compose.yaml [hunk 1]
```diff
@@ -1,36 +1,19 @@
...
         image: libero/browser:${IMAGE_TAG:-master}
-        depends_on:
-            - composer
     web:
-        image: nginx:1.15.2-alpine
+        image: nginx:1.15.5-alpine
         volumes:
-            - .docker/nginx.conf:/etc/nginx/conf.d/default.conf
+            - ./.docker/nginx.conf:/etc/nginx/conf.d/default.conf
+            - ./public/:/app/public/
```
Comment (giorgiosironi) [inline docker-compose.yaml]: if there was a sample file in here it would help validate the magical formulas to expose it through nginx

Comment (thewilkybarkid) [reply docker-compose.yaml]: Libero favicon?

Comment (giorgiosironi) [reply docker-compose.yaml]: Anything e.g.

**Comment by thewilkybarkid** [comment]
Build seems to be missing. 🤔

**Comment by giorgiosironi** [comment]
I think that Travis merges `master` into pull requests before building them. It may have seen no change in content after a merge of `master` here, although merged made in different places and conditions will produce different SHAs; difficult to say.

**Pull request merged**